PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskRawJetWithEP.h

```diff
@@ -1,10 +1,11 @@
-#ifndef ALIANALYSISTASKRAWJETWITHEP_H
-#define ALIANALYSISTASKRAWJETWITHEP_H
+
+#ifndef AliAnalysisTaskRawJetWithEP_H
+#define AliAnalysisTaskRawJetWithEP_H
 /**
- * \file AliAnalysisTaskRawJetWithEP.h
- * \brief Declaration of class AliAnalysisTaskRawJetWithEP
+ * \file AliAnalysisTaskEmcalJetSample.h
+ * \brief Declaration of class AliAnalysisTaskEmcalJetSample
  *
- * In this header file the class AliAnalysisTaskRawJetWithEP is declared.
+ * In this header file the class AliAnalysisTaskEmcalJetSample is declared.
```

Comment:
It's odd to change the name from you task name to the sample task name. Perhaps this is a copy and paste error?

Reply:
Dear Raymond, Thank you for review my code.
I modified the comment.